### DISCUSSION TITLE
Browserstack capabilities

### DISCUSSION THREAD

**Discussion by nichotined** [AppiumTestDistribution/AppiumTestDistribution#846] [Q&A] [answered]
Hi, I need to some browserstack capabilities such as `browserstack.gpsLocation` but i'm not really sure where to put it. 
Does anyone have tried it successfully? Please enlighten me 🙏

**Comment by saikrishna321** [comment, marked as answer]
@nichotined You can simply add any capabilities into the [capabilties.json](https://github.com/AppiumTestDistribution/AppiumTestDistribution/blob/master/caps/browserstack.json) file. ATD only parsers the given capabilities

**Comment by saikrishna321** [comment]
{
  "android": {
    "automationName": "UIAutomator2",
    "project": "ATD",
    "browserstack.gpsLocation": "40.730610,-73.935242",
    "app": {
      "local": "https://github.com/shridharkalagi/AppiumSample/raw/master/VodQA.apk",
      "cloud": "bs://a31eb3ea9002bb40a524ad76f7ce1ee44c06cefd"
    },
    "noSign": true
  },
  "iOS": {
    "platformName": "iOS",
    "app": {
      "simulator": "https://github.com/saikrishna321/AppiumTestDistribution/blob/master/apps/VodQAReactNative.zip?raw=true"
    },
    "deviceName": "iPhone",
    "wdaLaunchTimeout": 80000,
    "wdaConnectionTimeout": 80000,
    "preventWDAAttachments": true,
    "clearSystemFiles": true,
    "shouldUseSingletonTestManager": false,
    "simpleIsVisibleCheck": true,
    "maxTypingFrequency": 10
  },
  "cloud": {
    "android": [
      {
        "deviceName": "Google Pixel 3 XL",
        "osVersion": "9.0"
      },
      {
        "deviceName": "Samsung Galaxy S9 Plus",
        "osVersion": "9.0"
      }
    ]
  },
  "hostMachines": [
    {
      "isCloud": true,
      "machineIP": "hub-cloud.browserstack.com"
    }
  ]
}
